app.js

```diff
@@ -96,47 +93,47 @@ function getComputerValue() {
                 computerElements.paperElement.classList.add('item-visible');
 
                 if(appState.userSelectedItem === paper) {
-                    deadHeat();
+                    roundWinner('No one won this round...');
                 } else if(appState.userSelectedItem === rock) {
                     appState.points.computer++;
                     computerPoints.innerHTML = appState.points.computer;
-                    computerWonRound();
+                   roundWinner('Computer won this round...');
                 } else {
                     appState.points.user++;
                     userPoints.innerHTML = appState.points.user;
-                    userWonRound();
+                   roundWinner('You won this round...');
                 }
                 break;
             case 1:
                 appState.computerSelectedItem = rock;
                 computerElements.rockElement.classList.add('item-visible')
 
                 if(appState.userSelectedItem === rock) {
-                    deadHeat();
+                    roundWinner('No one won this round...');
                 } else if(appState.userSelectedItem === paper) {
                     appState.points.user++;
                     userPoints.innerHTML = appState.points.user;
-                    userWonRound();
+                   roundWinner('You won this round...');
                 } else {
                     appState.points.computer++;
                     computerPoints.innerHTML = appState.points.computer;
-                    computerWonRound();
+                   roundWinner('Computer won this round...');
                 }
                 break;
             default:
                 appState.computerSelectedItem = scissors;
                 computerElements.scissorsElement.classList.add('item-visible');
 
                 if(appState.userSelectedItem === scissors) {
-                    deadHeat();
+                    roundWinner('No one won this round...');
```

Comment:
let's try to keep with convention and use verbs for functions' names (like you would say: doSomething). For example, `handleGameOver`, `showEndMessage`, etc.